test/wdio/shadow-dom-slot-nested/cmp.test.tsx

```diff
@@ -0,0 +1,114 @@
+import { Fragment, h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+const CSS = `main {
+  color: blue;
+  font-weight: bold;
+}`;
+
+describe('shadow-dom-slot-nested', () => {
+  beforeEach(async () => {
+    render({
+      template: () => (
+        <>
+          <main>
+            main content
+            <shadow-dom-slot-nested-root></shadow-dom-slot-nested-root>
+          </main>
+          <style>{CSS}</style>
+        </>
+      ),
+    });
+  });
+
+  it('renders children', async () => {
+    const elm = $('main');
+    await expect(elm.getCSSProperty('color')).toMatchInlineSnapshot(`
+      {
+        "parsed": {
+          "alpha": 1,
+          "hex": "#0000ff",
+          "rgba": "rgba(0,0,255,1)",
+          "type": "color",
+        },
+        "property": "color",
+        "value": "rgba(0,0,255,1)",
+      }
+    `);
+
+    const cmp = $('shadow-dom-slot-nested-root');
+    const section = cmp.shadow$('section');
+    await expect(section.getCSSProperty('color')).toMatchInlineSnapshot(`
+      {
+        "parsed": {
+          "alpha": 1,
+          "hex": "#008000",
+          "rgba": "rgba(0,128,0,1)",
```

Comment:
It looks like Firefox would prefer this to be `rgb` values instead of `rgba` 😢

Reply:
bummer .. updated test to validate against hex values